### PR TITLE
Add blogwriter css

### PR DESCRIPTION
**Pull request by Third9**
Add blogwriter css

### CODE REVIEW

File: files/blogwriter/info.ini [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author = "third9"
+github = "https://github.com/Third9"
```
Comment (megawac) [inline files/blogwriter/info.ini]: Hi, this should be the source code of the library (e.g. a github repo)

File: files/blogwriter/info.ini [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+homepage = "http://blogwriter.herokuapp.com"
+mainfile = "blogwriter.css"
```
Comment (megawac) [inline files/blogwriter/info.ini]: Can you provide a minified file

**Comment by Third9** [comment]
Update Info.ini 
and Add minified file

**Comment by megawac** [comment]
@jsdelivrbot squash & merge

File: files/blogwriter/info.ini (hunk 2 above)
Comment (Third9) [inline files/blogwriter/info.ini]: sure i add minified file and info.ini

**Pull request closed without merging**